docs/dictionary/command/send.lcdoc

```diff
@@ -38,21 +38,21 @@ message: An expression that evaluates to a message name, possibly including para
 object: Any object reference. If you don't specify an object, the message is set to the object whose handler contains the send command. If you specify a time, you must also specify an object.
 time (integer): If you don't specify a unit of time, the default of ticks is used.
 
-The result: If you specify a <time>, the <message> is sent to the <object> after the specified time has elapsed. The <message> is added to the <pendingMessages> <function>. The ID of the <message> is <return|returned> by the <result> <function>. To cancel the <message>, use the <cancel> <command> with this ID.
+The result: The ID of the <message> in the <pendingMessages> <function> queue.
 
 Description:
-Use the <send> <command> to override the normal <message path>, or to delay a <command> until a specified time. 
+Use the <send> <command> to override the normal <message path>, or to delay a <command> until after a specified time.
 
-The <message> can be either the name of a <handler> or a <LiveCode> <statement>. If the <message> is a <literal string> containing more than one <word> (for example, if it includes <parameter|parameters>, or if it is a multi-word command), it must be enclosed in <double quote|double quotes>. 
+The <message> can be either the name of a handler or a <LiveCode> <statement>. If the <message> is a <literal string> containing more than one <word> (for example, if it includes <parameter|parameters>, or if it is a multi-word command), it must be enclosed in <double quote|double quotes>.
```

Comment:
The angle brackets around `<handler>` here are to provide a link to the handler glossary entry- they shouldn't be removed.